src/components/screens/DocsScreen/DocsScreen.tsx

```diff
@@ -79,17 +82,37 @@ function DocsScreen({ data, pageContext }) {
       frontmatter: { title },
     },
   } = data;
-  const { tocItem, nextTocItem } = pageContext;
+  const { features } = useSiteMetadata();
+  const { framework, slug, tocItem, nextTocItem } = pageContext;
   const CodeSnippetsWithPageContext = useMemo(() => {
     return (props) => <CodeSnippets pageContext={pageContext} {...props} />;
   }, []); // TODO: Make this dependent on the framework when it is available
 
+  const FrameworkSupportTableWithFeaturesAndCurrentFramework = useMemo(() => {
+    return ({ frameworks }) => (
+      <FrameworkSupportTable
+        frameworks={frameworks}
+        currentFramework={framework}
+        features={features}
+      />
+    );
+  }, []);
```

Comment:
I'm not sure if the component is reused when you switch framework? If so, we should add `framework` to the dependencies of this `useMemo`.

Reply:
I'll handle this in an upcoming PR. It likely is reused.